modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionContext.java

```diff
@@ -132,6 +137,9 @@ public ExecutionContext(
         this.originatingNodeName = originatingNodeName;
         this.txAttributes = txAttributes;
         this.timeZoneId = timeZoneId;
+        this.inBufSize = inBufSize < 0 ? Commons.IN_BUFFER_SIZE : inBufSize;
```

Comment:
just suppose - this configurable param is available from cluster setting (i hope this was the main goal for change) **-ea** is not activated in production and someone want`s to set it into "0" ?

Reply:
Or maybe '0' should means no-buffering?
I prefer having magic constant like `-1`, which mean not-specified, than thinking what does '0' could means.